extensions/2.0/Vendor/EXT_geopose/0.0.0/README.md

```diff
@@ -0,0 +1,68 @@
+<!-- omit in toc -->
+# EXT_geopose
```

Comment:
The extension intuitively makes sense but there should be a sentence about how this extension interacts with the glTF scene graph and [coordinate system](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#coordinate-system-and-units).

* The geopose transform would be applied last after the node hierarchy
* +z would be facing east, +x would be facing north, and +y would be facing up. I put together a sandcastle to confirm: [Sandcastle](https://sandcastle.cesium.com/#c=bZJbb9QwEIX/ipWnrBSc3QICbdMVaLeFBwoVW/EUCXmT2WSEL9HY2bYg/jt2nL2Vvtln5pz5Js5OENshPACxK6bhgS3BYq/4j0FLy6Qa7kujnUANVCaTy1JHB7cVaODK1ODNo3EdtFsv8fXy+uv1xcq3e4Mf04KoUTe+dcqnl1Hr0FXtmUJGyjNBGt2g64cZr96d6MLt5TdvD2oL2LTOa7NpbB3ljo6InyPIXZj93Y/jWzJqBQ0B2HSkzCJaNvBMxnBD2KA+Bi0FOX8S+vVZxIE4O0BmI9g+KXwzeSsc4eMx7p6EtltDyvL2GeK9ucFHqG9IKEgjRhaWmhw2HBJ91tnTdIQKHe7AclHXaanZflZ4obj4J+m26Z9QYqwnOWdlwnm+FqqTsBJO5EOyzaPxI9J4+umPvJGbMsmi+WSn+ellX0aNqld3fg+5xt8wZ7OL90Pt76TUYZEkSwrrniQsooOxD6g6Qy5wpR7KgYcS/pPnm776BY5X1gZjaC3yU2tR445hffXC/8sqKaz1lW0vB5AyWRS57//PKs3wCN92QFI8hbZ2tvgSRc55kfvry05njNwIepb8Dw)

![image](https://user-images.githubusercontent.com/915398/117360304-9c88e500-ae86-11eb-902f-2e100d6e072a.png)

It would be helpful to include a snippet of that CesiumJS code as a reference implementation in an appendix. Also maybe link directly to [`Transforms.headingPitchRollToFixedFrame`](https://github.com/CesiumGS/cesium/blob/dacc86c6a32f1ece62d66ac20b6ea5ff6f040ad0/Source/Core/Transforms.js#L385-L412).

Reply:
By the way I am happy to get on a call, or maybe even invite you as a guest to an upcomming Standards Working Group meeting where you can provide input on the current draft from your perspective. You can reach me on linked in or twitter https://twitter.com/Jan_Erik_Vinje  https://www.linkedin.com/in/janerikvinje/

Reply:
@janerivi we are deeply interested in aligning efforts.  I apologize for the lack of context in the pull request.  @LisaBosCesium will be in touch.